database/src/main/resources/db/migration/R__GTFS_query.sql

```diff
@@ -370,23 +374,23 @@ BEGIN
          trip_chg := gtfs_trip_changes(ROW(row."d1-package-id", row."d1-trip")::"gtfs-package-trip-info",
                                        ROW(row."d2-package-id", row."d2-trip")::"gtfs-package-trip-info",
                                        first_common_stop);
-         IF trip_stop_seq_changes IS NULL THEN
-           trip_stop_seq_changes := int4range(trip_chg."trip-stop-sequence-changes",
-                                              trip_chg."trip-stop-sequence-changes", '[]');
-         ELSE
-           trip_stop_seq_changes := range_merge(trip_stop_seq_changes,
-                                                int4range(trip_chg."trip-stop-sequence-changes",
-                                                          trip_chg."trip-stop-sequence-changes", '[]'));
+
+         IF ((trip_stop_seq_changes_lower IS NULL) or (trip_chg."trip-stop-sequence-changes" < trip_stop_seq_changes_lower)) THEN
```

Comment:
NULL check could be omitted by initializing variable to `0`.
Will improve this in next PR.